tests/suites/test_suite_ssl.data

```diff
@@ -3445,3 +3445,185 @@ tls13_srv_max_early_data_size:TEST_EARLY_DATA_HRR:3:3
 
 TLS 1.3 srv, max early data size, HRR, 98, wsz=49
 tls13_srv_max_early_data_size:TEST_EARLY_DATA_HRR:97:0
+
```

Comment:
I suggest placing the new tests in a separate `.data` file. `test_suite_ssl.data` [is already huge](https://github.com/Mbed-TLS/mbedtls/issues/4967). Creating a new `.data` file for an existing `.function` file is trivial, it doesn't need to be recorded in any build script.